spec/decorators/course_decorator_spec.rb

```diff
@@ -374,4 +378,44 @@
       end
     end
   end
+
+  describe "#main_address" do
+    context "with a full address" do
+      let(:provider) do
+        build(:provider,
+              accredited_body?: false,
+              website: "www.acmescitt.com",
+              address1: "Building 64",
+              address2: "32 Copton Lane",
+              address3: "Bracknel",
+              address4: "Berkshire",
+              postcode: "NXT STP")
+      end
+
+      it "can return a full address" do
```

Comment:
`it "returns...` would be more consistent with the rest of the file here and 410